<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="User-PC" name="assignment_Student.Attempt_test" tests="1" failures="1" timestamp="19 Dec 2016 10:15:33 GMT" time="4043.068" errors="0">
  <testcase name="Exicute" time="4043.068" classname="assignment_Student.Attempt_test">
    <failure type="java.lang.AssertionError" message="Unable attempt MCQ Question">
      <![CDATA[java.lang.AssertionError: Unable attempt MCQ Question
	at org.testng.Assert.fail(Assert.java:94)
	at assignment_Student.Student_MCQ_Test.MCQ_Test(Student_MCQ_Test.java:336)
	at assignment_Student.Attempt_test.attemptTest(Attempt_test.java:210)
	at assignment_Student.Attempt_test.Exicute(Attempt_test.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- Exicute -->
</testsuite> <!-- assignment_Student.Attempt_test -->
